File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.cpp
// User function Template for C++
class Solution {
  public:
    pair<int, int> getMinMax(vector<int> arr) {
        // code here
        int largest = INT_MIN;
        int smallest = INT_MAX;
        for(int i=0;i<arr.size();i++){
            if(arr[i]>largest){
                largest = arr[i];
            }
            if(arr[i]<smallest){
                smallest = arr[i];
            }
        }
        return {smallest,largest};
    }
};

File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.java
class Solution {
    public ArrayList<Integer> getMinMax(int[] arr) {
        // code Here
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        
        ArrayList l1 = new ArrayList();
        
        for(int i=0;i<arr.length;i++)
        {
            if(arr[i]>max)
            {
                max = arr[i];
            }
            if(arr[i]<min)
            {
                min = arr[i];
            }
        }
        
        l1.add(min);
        l1.add(max);
        
        return l1;
    }
}
